OasisDEX/oasis-borrow
# New vault layout in protection tabs

# [New vault layout in protection tabs](https://app.shortcut.com/oazo-apps/story/4181/implement-new-vault-details-section-in-protection-tab)
  
## Changes 👷‍♀️

Based on https://github.com/OasisDEX/oasis-borrow/pull/982.

Vault protection tab is rewritten into new layout according to figma: https://www.figma.com/file/S6pnwbeRY6T8hfNQKxzp7K/Components?node-id=504%3A2130

- Buttons in header sections are out, they will be moved into new sidebar.
- Modals with details for each card weren't changed in any way according to acceptance criteria from previous story regarding components used here: https://app.shortcut.com/oazo-apps/story/3856/redesign-update-detail-cards
  
## How to test 🧪

`AutomationBasicBuyAndSell` feature flag is needed to see any changes, `Automation` flag must be on to see Protection tab.

Additionally, similar to https://github.com/OasisDEX/oasis-borrow/pull/982, sonar cloud and codecov code smells might have problem with this PR, but because we are using feature flag some of the code HAD to be duplicated. All of it is going to be removed as soon as we can remove feature flag along with duplicated lines or entire files.

And last thing, don't mind amount of commits, focus on files changed. This branch was started not from the `dev`, but from branch mentioned before, https://github.com/OasisDEX/oasis-borrow/pull/982, so it shows changes that are already merged to `dev`.
